src/frontend/login_page/components/LoginPage.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { Link } from 'react-router-dom';
+import CSS from 'csstype';
+
+export interface LoginPageState {
+    username: string,
+    password: string,
+    submitted: boolean
+}
+
+interface LoginPageProp {
+
+}
+
+class LoginPage extends React.Component <LoginPageProp, LoginPageState>{
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            username: '',
+            password: '',
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
I don't think this is needed if you are using arrow functions. (Maybe I was wrong)